Global.h

```diff
@@ -685,6 +685,19 @@ namespace vstd
 		return a + (b - a) * f;
 	}
 
+	template<typename Floating>
+	bool isAlmostZero(const Floating & value)
+	{
+		constexpr Floating epsilon(0.00001);
```

Comment:
Wouldn't https://en.cppreference.com/w/cpp/types/numeric_limits/epsilon be better?

Reply:
No, that one is too small.
Working with float is *fun*. For example, simple expression like `a*b/c` can be compiled as `a/c*b`, potentially leading to rounding error. And more operations you have - then more these rounding errors can accumulate. In case of 1 operation, value from numeric_limits might be suitable, but when you have potentially large number of operations you want to have much larger epsilon